test/common.js

```diff
@@ -170,6 +170,24 @@ test('broken links still expand', t => {
   t.deepEqual(result, ['resources/badlink']);
 });
 
+test('empty array', t => {
+  const result = common.expand([]);
+  t.falsy(shell.error());
+  t.deepEqual(result, []);
+});
+
+test('empty string', t => {
+  const result = common.expand(['']);
+  t.falsy(shell.error());
+  t.deepEqual(result, ['']);
+});
+
+test('non-string', t => {
+  const result = common.expand([5]);
+  t.falsy(shell.error());
```

Comment:
No need to check `shell.error()`, it's never set by this function (it's internal). Maybe just file a bug to clean up these tests later.